javalib/src/main/scala/java/net/WindowsPlainSocketImpl.scala

```diff
@@ -109,10 +109,13 @@ private[net] class WindowsPlainSocketImpl extends AbstractPlainSocketImpl {
   ): Unit = {
     val mode = stackalloc[Int]
     if (blocking)
-      !mode = 1
-    else
       !mode = 0
-    ioctlSocket(fd.handle, FIONBIO, mode)
+    else
+      !mode = 1
```

Comment:
Why did the value of `mode` get inverted?

Reply:
Previous usage of mode was incorrect, I have misinterpreted the documentation of ioctlSocket. 

Fragment from Microsoft [documentation of ioctlSocket](https://docs.microsoft.com/en-us/windows/win32/api/winsock/nf-winsock-ioctlsocket):
```
//-------------------------
// Set the socket I/O mode: In this case FIONBIO
// enables or disables the blocking mode for the 
// socket based on the numerical value of iMode.
// If iMode = 0, blocking is enabled; 
// If iMode != 0, non-blocking mode is enabled.

iResult = ioctlsocket(m_socket, FIONBIO, &iMode);

```